internal/bminventory/inventory_test.go

```diff
@@ -71,6 +72,12 @@ func getTestAuthHandler() auth.AuthHandler {
 	return *auth.NewAuthHandler(fakeConfigDisabled, nil, getTestLog().WithField("pkg", "auth"), nil)
 }
 
+func getPullSecretValidator() *validations.PullSecretValidator {
```

Comment:
Maybe it should be a mock? unit tests usually validate only the package that it tests.

Reply:
You're right, will change that.